### ISSUE TITLE
Deprecated?

### ISSUE THREAD

**Issue by rilakuma123** [OroArmor/Oro-Config#8]
Is this project deprecated? I didn't see a notice or anything.

**Comment by rilakuma123** [comment]
Oh https://github.com/OroArmorModding/Oro-Config/pull/7#issuecomment-1700416028

**Comment by OroArmor** [comment]
Yeah its effectively dead. I have no plans to maintain it, but if others do i will keep it alive for them
